crates/ruma-common/tests/events/message.rs

```diff
@@ -354,3 +354,45 @@ fn emote_event_deserialization() {
             && unsigned.is_empty()
     );
 }
+
+#[test]
+#[cfg(feature = "unstable-msc3554")]
+fn lang_serialization() {
+    use ruma_common::events::message::{MessageContent, Text};
+
+    let content = MessageContent::from(vec![
+        assign!(Text::plain("Bonjour le monde !"), { lang: Some("fr".into()) }),
```

Comment:
Why the space before the exclamation mark? 😛

Reply:
French typography!

Reply:
Whaaaat?

Reply:
Non-breaking space before some punctuation marks. It's not a well-known rule (also in the wild but anyone having to write something "serious" should respect it. However I think mobile OSes apply it. It's also not a very convenient character to find so most people use a regular space.